R/model.R

```diff
@@ -563,7 +564,8 @@ predict.keras.engine.training.Model <- function(object, x, batch_size=NULL, verb
   # args
   args <- list(
     batch_size = as_nullable_integer(batch_size),
-    verbose = as.integer(verbose)
+    verbose = as.integer(verbose),
+    callbacks = normalize_callbacks(NA, callbacks)
```

Comment:
Does this need to be conditional on Keras version or is this supported in all versions >= 2.0?

Reply:
Yes. At the moment it will only work with tk.keras and tf >= 2.0. This has been merged into keras master and should probably appear in the next release. I'll fix this to work on the other versions.